README.md

```diff
@@ -26,16 +24,36 @@ TODO: Write usage instructions here
 
 ## Development
 
-After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake test` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
+After checking out the repo, run `bin/setup` to install dependencies.
+Then, run `rake test` to run the tests. You can also run `bin/console` for
+an interactive prompt that will allow you to experiment.
 
-To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb`, and then run `bundle exec rake release`, which will create a git tag for the version, push git commits and tags, and push the `.gem` file to [rubygems.org](https://rubygems.org).
+To install this gem onto your local machine, run `bundle exec rake install`.
+To release a new version, update the version number in `version.rb`, and
+then run `bundle exec rake release`, which will create a git tag for the
+version, push git commits and tags, and push the `.gem` file to [rubygems][]
 
 ## Contributing
 
-Bug reports and pull requests are welcome on GitHub at https://github.com/[USERNAME]/pdc. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [Contributor Covenant](http://contributor-covenant.org) code of conduct.
+Bug reports and pull requests are welcome on GitHub at
+[product-definition-center/pdc-ruby-gem][pdc-github] . This project is intended to be a safe,
+welcoming space for collaboration, and contributors are expected to adhere to
+the [Contributor Covenant][coc] code of conduct.
 
 
 ## License
 
-The gem is available as open source under the terms of the [MIT License](http://opensource.org/licenses/MIT).
+The gem is available as open source under the terms of the [MIT License][mit-license]
+
+## Credits
+This gem is heavily influenced by the [spyke][] gem which requires
+`activesupport 4.0` or above where as the `pdc` gem needed to run on
+`activesupport 3.2.22`. If you using `activesupport 4` or above and is looking
+for json api to ORM framework please consider using [spyke][] instead.
+
```

Comment:
Not a conflict but there are new features and api incompatibility between 3.2 and 4.0. Spyke relies on those, thus would fail to run against 3.2. In PDC gem's case we can't move to activesupport 4 until the rails application that uses this gem upgrades to rails 4 or 5.